x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/investigation_section.tsx

```diff
@@ -28,6 +28,7 @@ export const InvestigationSection = memo(() => {
   const { dataFormattedForFieldBrowser, getFieldsData, investigationFields, scopeId } =
     useDocumentDetailsContext();
   const eventKind = getField(getFieldsData('event.kind'));
+  const ancestorIndex = getField(getFieldsData('signal.ancestors.index')) ?? '';
```

Comment:
`<HighlightedFields />` seems to accept `undefined` as well, no need to default to an empty string

Reply:
Yeah, but `getField` returns `string | null` which is not compatible with `string | undefined`, that's why I defaulted 😄 
Let me know if you have a better approach in mind

Reply:
Sounds good, let's keep it!